core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -340,12 +339,10 @@ class HttpGoogleServicesDAO(
         val patcher = buckets.patch(bucketName, new Bucket().setLifecycle(lifecycle))
```

Comment:
In the comments here (L334) the second link is broken - we should remove (and maybe replace?)

Reply:
done